### PR TITLE
[feature](statistics)support statistics for iceberg/paimon/hudi table

### PR DESCRIPTION
**Pull request by wuwenchi**
## Proposed changes

We can use `analyze table <iceberg/paimon/hudi _table>` to collect statistical information.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by wuwenchi** [comment]
run buildall

**Comment by wuwenchi** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/statistics/CommonAnalysisTask.java [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+import org.apache.logging.log4j.Logger;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+import java.util.Set;
+
+public class CommonAnalysisTask extends BaseAnalysisTask {
```
Comment (Jibing-Li) [inline fe/fe-core/src/main/java/org/apache/doris/statistics/CommonAnalysisTask.java:38]: Is this CommonAnalysisTask for all external tables? How about rename it to ExternalAnalysisTask or something similar? I feel CommonAnalysisTask is kind of confusing, looks no difference with BaseAnalysisTask

**Comment by wuwenchi** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/statistics/CommonAnalysisTask.java (hunk 1 above)
Comment (wuwenchi) [reply fe/fe-core/src/main/java/org/apache/doris/statistics/CommonAnalysisTask.java]: done

**Comment by wuwenchi** [comment]
run buildall

**Comment by wuwenchi** [comment]
run buildall

**Comment by wuwenchi** [comment]
run buildall

**Review by Jibing-Li — APPROVED** [review]
LGTM

**Comment by wuwenchi** [comment]
run buildall

**Comment by wuwenchi** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**